migrations/2022-11-27-180447_custom_emojis/up.sql

```diff
@@ -0,0 +1,16 @@
+create table custom_emoji (
+  id serial primary key,
+  local_site_id int references local_site on update cascade on delete cascade not null,
+  shortcode varchar(128) not null,
+  image_url text not null,
+  alt_text text null,
+  category text not null,
+  published timestamp without time zone default now() not null,
+  updated timestamp without time zone
+);
+
+create table custom_emoji_keyword (
+  id serial primary key,
+  custom_emoji_id int references custom_emoji on update cascade on delete cascade not null,
+  keyword varchar(128) not null
```

Comment:
What's the difference between a keyword and a shortcode?

Reply:
shortcode is the unique signifier for displaying the emoji, i.e. :my-customicon:

keywords are extra identifiers to aid in searching through to find emoji.

Reply:
Gotcha. The keyword should also probably be unique. We also *could* add a lowercase unique constraint, but I'd just make sure its lowercase before you insert it into the DB.